helm/bankapp/values.yaml

```
@@ -0,0 +1,54 @@
+# namespace 
+namespace: bankapp-namespace
+
+# name of configmap, database and db user.
+configmap:  
+  name: bankapp-config
+  data:
+      MYSQL_DATABASE: BankDB
```

yamllint (1.35.1)

[warning] wrong indentation: expected 4 but found 6

(indentation)